vllm/envs.py

```diff
@@ -268,6 +268,11 @@ def maybe_convert_int(value: Optional[str]) -> Optional[int]:
     "VLLM_API_KEY":
     lambda: os.environ.get("VLLM_API_KEY", None),
 
+    # Whether to log responses from API Server
+    "LOG_API_SERVER_RESPONSE":
```

Comment:
Can we call it: `VLLM_DEBUG_LOG_API_SERVER_RESPONSE`

Reply:
Sure, I can update that.